src/scripts/prepareCiceroInput.pl

```diff
@@ -33,46 +33,63 @@
 	'v|version'		=> \$version,
 );
 
+## Perhaps someday someone will come back and finish the pod
 pod2usage(-verbose=>2) if($man or $usage);
 pod2usage(1) if($help or $version );
 
+die "out_dir is not specified" unless defined($out_dir) && $out_dir ne '';
 mkdir $out_dir if(!-d $out_dir);
 
-my $conf; 
+my $conf;
 if (&TdtConfig::findConfig("genome", $genome)){
-	$conf = &TdtConfig::readConfig("genome", $genome); 
+	$conf = &TdtConfig::readConfig("genome", $genome);
 }
 else{
-	croak "no config"; 
+	croak "no config";
+      }
+
+sub validate_filename {
+  ## Called with file name and 'descriptor'
+  my($fname, $fdesc) = @_;
+  if (!defined($fname) || $fname eq '') {
+    die "No $fdesc has been defined";
+  } elsif (!-f $fname) {
+    die "$fdesc file $fname not found"
+  }
 }
 
 my $black_list_file = $conf->{'BLACKLIST_GENES'};
-open(my $BLK, "<", "$black_list_file");
+validate_filename($black_list_file, 'blacklist');
```

Comment:
I don't think we need this. If you rebase, all of these `open` commands should have a `|| die ` check to ensure that the file exists.